src/Evolution/Executables/ScalarWave/EvolveScalarWave.hpp

```diff
@@ -60,6 +61,13 @@ struct EvolutionMetavars {
   using const_global_cache_tag_list = tmpl::list<analytic_solution_tag>;
   using domain_creator_tag = OptionTags::DomainCreator<Dim, Frame::Inertial>;
 
+  using Redum = Parallel::ReductionDatum<double, funcl::Plus<>,
+                                         funcl::Sqrt<funcl::Divides<>>,
+                                         std::index_sequence<1>>;
+  using reduction_data_tags = tmpl::list<observers::Tags::ReductionData<
+      Parallel::ReductionDatum<double, funcl::AssertEqual<>>,
```

Comment:
according to your commit order, AssertEqual isn't available yet...

Reply:
Yea, my order is definitely what you get from `git log` (so 1. is the last, sorry)